presto-hive-common/src/main/java/com/facebook/presto/hive/HdfsContext.java

```diff
@@ -15,6 +15,7 @@
 
 import com.facebook.presto.spi.ConnectorSession;
 import com.facebook.presto.spi.security.ConnectorIdentity;
+import org.apache.hadoop.fs.Path;
```

Comment:
Let's avoid having the `Path` in the interface, as it doesn't have very clear semantics. I would recommend to represent Path as a simple string, unless there's a very good reason not to.